examples/bq-email-exports/source_1/main.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Cloud Function to run the specified query on a BigQuery anonymous table.
+Triggered by Cloud Scheduler.
+"""
+
+from google.cloud import bigquery
+
+
+def main(event, context):
+    """Entrypoint for Cloud Function"""
+
+    # Set variables
+    query = "SELECT CONCAT('https://stackoverflow.com/questions/'," \
```

Comment:
It'd be best if this query text was treated as a config file rather than part of the python source. 
For example could this query text be stored in GCS? the cloud function could be deployed w/ a `GCS_QUERY_PATH` environment vauriable from which it download the query text. This will make it much easier to reuse this because you just reuse the same source over and over again w/ different env var. Currently to reuse this you have to edit the python source each time.